index.js

```diff
@@ -42,6 +46,7 @@ module.exports = {
     "max-line-length": [ 80, {
       ignore: "non-comments",
     } ],
+    "max-nesting-depth": 3,
```

Comment:
I think 3 is a good limit yeah.

Why might we want to have another stylelint plugin for nesting? Wouldn't it make sense to keep it all in one place?